classifiers/reference_quality/tiktoken_length_classifier/code_snippet_refinery.md

```diff
@@ -0,0 +1,19 @@
+```python
+import tiktoken 
+
+ATTRIBUTE: str = "text" # only text attributes
+ENCODING_NAME: str = "cl100k_base"
+
+
+def tiktoken_length_classifier(record):
+    encoding = tiktoken.get_encoding(ENCODING_NAME)
```

Comment:
we can move the definition of the encoding outside of the method, so we do not need to do this for each record in a labeling function

Reply:
Yes, moved this to outside of the function.